dotnet-desktop-guide/framework/winforms/keyboard-input-in-a-windows-forms-application.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: Keyboard input
+description: Learn about and understand how keyboard input works with Windows Forms so that your application can respond to specific key presses.
```

Comment:
```suggestion
description: Learn about how keyboard input works with Windows Forms so that your application can respond to specific key presses.
```